lib/matplotlib/collections.py

```diff
@@ -2047,6 +2046,40 @@ def set_paths(self):
         self._paths = self._convert_mesh_to_paths(self._coordinates)
         self.stale = True
 
+    def set_array(self, A):
```

Comment:
This is public API, so it needs a docstring.

Reply:
Ah, will do.

Reply:
I'm not really sure how to structure this docstring, please make any changes if required

Reply:
Well, it would have inherited a docstring from `ScalarMappable`, I think.
https://github.com/matplotlib/matplotlib/blob/63261cc23e145c445a9907d84a9db2b1ecd878eb/lib/matplotlib/cm.py#L363-L373
Though some of that needs to be overridden anyway, so it's good to write one here.